src/field/field255.rs

```diff
@@ -289,6 +289,24 @@ impl Default for Field255 {
     }
 }
 
+impl TryFrom<Field255> for u64 {
+    type Error = FieldError;
+
+    fn try_from(elem: Field255) -> Result<u64, FieldError> {
+        let mut le_bytes = [0; 32];
+        fiat_25519_to_bytes(&mut le_bytes, &elem.0);
+        let mut is_zero = Choice::from(1);
+        for byte in &le_bytes[8..] {
+            is_zero &= !Choice::from(*byte);
```

Comment:
This constructor should only be used with `0u8` or `1u8`, changing the "too_big" test to use `[2; 32]` hits a debug assertion inside subtle. Instead, we should use `<u8 as ConstantTimeEq>` methods.